src/script/script.cpp

```diff
@@ -6,6 +6,7 @@
 
 #include "script.h"
 #include "interpreter.h"
+#include "util.h"
```

Comment:
Not used after you switched from `DbgAssert` to `assert`

Reply:
agreed. done.